ir/src/symbol_table.rs

```diff
@@ -43,6 +47,10 @@ pub(super) struct SymbolTable {
     /// Vector in which index is trace segment and value is number of columns in this segment
     segment_widths: Vec<u16>,
 
+    /// Number of random values. For array initialized in `rand: [n]` form it will be `n`, and for
+    /// `rand: [a, b[n], c, ...]` it will be length of the array.
+    random_values_num: u16,
```

Comment:
The convention we usually use is to put `num` in front. So, this would be `num_random_values`.